content/pt-br/docs/reference/issues-security/issues.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Rastreador de Issue Kubernetes
+weight: 10
+aliases: [/pt-br/cve/,/pt-br/cves/]
+---
+
+Para reportar um problema de segurança, siga [processo de divulgação de issues do Kubernetes](/docs/reference/issues-security/security/#report-a-vulnerability).
+
+O trabalho no código do Kubernetes e os problemas de segurança podem ser encontrados usando [ GitHub Issues ](https://github.com/kubernetes/kubernetes/issues/).
```

Comment:
I'd use "issues do GitHub" here instead of "GitHub Issues". It would also be better to remove the spaces inside the square bracket, as these are getting rendered.